spec/javascripts/surveys-spec.js

```diff
@@ -692,9 +692,9 @@ describe('Surveys', function () {
         var testSurvey = {
           startTime: new Date('July 5, 2016').getTime(),
           endTime: new Date('July 10, 2016 23:50:00').getTime(),
-          activeWhen: function () { return false },
           url: 'example.com/small-survey'
         }
+        spyOn(surveys, 'activeWhen').and.returnValue(false)
```

Comment:
Now that a function isn't allowed for `activeWhen` on surveys this is a good way to fix this test, but I think we should do the same to the sibling test above: "includes the survey when the activeWhen function returns true".  That test is passing "just because" and it'd be better to be more explicit.